src/lib/UniversalDiscovery/Event/Subscriber/ObjectRelationAllowedContentTypes.php

```diff
@@ -54,7 +54,9 @@ public function onUdwConfigResolve(ConfigResolveEvent $event): void
                 ? null
                 : $intersection;
         } else {
-            $config['allowed_content_types'] = $context['allowed_content_types'];
+            $config['allowed_content_types'] = empty($context['allowed_content_types'])
```

Comment:
Are you sure about `empty` here? `[]` is valid value (meaning context disallow any content type) and it will resolve to `null` now (meaning every content type is allowed).

Reply:
In the Back Office there are two valid values for `allowed_content_types`:
- `[]` resulting in no available CT to select in UDW,
- `null` resulting in every CT available to select in UDW.

In the FieldType edition view by default, there is no CT selected which stays in contradiction with the label `Allowed Content Types`. So in short: no allowed content type selected means all are available. That was the case in 2.5 at least, so I wanted to keep it consistent. That's why we need `null` value in this case.

Reply:
If I read this correctly, frontend part sends `[]` when nothing is selected, but then it want's `null` for that same behaviour?

Shouldnt it be more distinct on UDW part?
@dew326 
@Nattfarinn

Reply:
Confirmed, this is OK because of how FieldType works.